src/PropertyBuilder/AttributeBuilder.php

```diff
@@ -49,22 +49,25 @@ function (ProductInterface $product, Document $document): void {
 
     private function resolveProductAttributes(ProductInterface $product, Document $document): void
     {
+        /** @var ProductAttributeValue $productAttribute */
         foreach ($product->getAttributes() as $productAttribute) {
             $attribute = $productAttribute->getAttribute();
-            if (!$attribute) {
+            if (null === $attribute) {
```

Comment:
This seems to be a logic change. I thing author could potentially intent to check !.